pages/Getting Started/Installation.md

```diff
@@ -174,14 +174,17 @@ Hyprland and related are in the default repository:
 
 {{% details title="Ubuntu*" closed="true" %}}
 
-Hyprland made it into the universe repos and can be installed with 
+Hyprland made it into the Ubuntu 24.10 Oracular Oriole universe repo and can be installed with 
 ```bash
 sudo add-apt-repository universe && sudo apt-get update && sudo apt-get install -y hyprland
 ```
-Build dependencies:
+> [!NOTE]
+> Above is for Ubuntu 24.10 (Unreleased) version
+
+For installing Hyprland from Source, install first the dependencies below:
 
 ```bash
-sudo apt-get install -y meson wget build-essential ninja-build cmake-extras cmake gettext gettext-base fontconfig libfontconfig-dev libffi-dev libxml2-dev libdrm-dev libxkbcommon-x11-dev libxkbregistry-dev libxkbcommon-dev libpixman-1-dev libudev-dev libseat-dev seatd libxcb-dri3-dev libegl-dev libgles2 libegl1-mesa-dev glslang-tools libinput-bin libinput-dev libxcb-composite0-dev libavutil-dev libavcodec-dev libavformat-dev libxcb-ewmh2 libxcb-ewmh-dev libxcb-present-dev libxcb-icccm4-dev libxcb-render-util0-dev libxcb-res0-dev libxcb-xinput-dev xdg-desktop-portal-wlr libtomlplusplus3
```

Comment:
Is there `xdg-desktop-portal-hyprland` in the Ubuntu repos?

Reply:
no, xdph still not in ubuntu repos....

But the reason I removed the xdpw is because you can still build Hyprland even without it.

Just need to download and install and install xdph from source

Reply:
Then that should also be mentioned. Otherwise, users will complain about screensharing not working.

Reply:
ok done